client/superdesk.config.js

```diff
@@ -4,29 +4,171 @@
  * working directory, but other files may also be specified using relative paths with
  * the SUPERDESK_CONFIG environment variable or the grunt --config flag.
  */
-module.exports = function (grunt) {
-  return {
-    apps: ["superdesk-planning"],
-    importApps: ["../index", "superdesk-planning"],
-    defaultRoute: "/workspace/personal",
-    features: {
-      swimlane: { defaultNumberOfColumns: 4 },
-      noTakes: true,
-      planning: true,
-    },
-    validatorMediaMetadata: {
-      headline: { required: true },
-      alt_text: { required: true },
-      description_text: { required: true },
-      copyrightholder: { required: false },
-      byline: { required: false },
-      usageterms: { required: false },
-      copyrightnotice: { required: false },
-    },
-    workspace: {
-      planning: true,
-      assignments: true,
-    },
-    spellcheckers: { fi: "stt_fin" },
-  };
+module.exports = function(grunt) {
+    return {
+    	/* enable modules */
+        apps: [
+            'superdesk-planning', 
+            'superdesk.analytics' 
+        ],
+        importApps: [
+            '../index',
+            'superdesk-planning',
+            'superdesk-analytics',
+        ],
+        workspace: {
+            planning: true,
+            assignments: true,
+            analytics: true,
+        },
+        
+        /* landing page after login */
+        defaultRoute: '/workspace/monitoring',
+        server: {
+            url: "http://172.16.109.15:5000/api",
+            ws: "ws://172.16.109.15:5100/ws"
```

Comment:
hardcoding those would probably break our setup, it should be able to read it from `SUPERDESK_URL` and `SUPERDESK_WS_URL` env variables